src/beautifiers/rubocop.coffee

```diff
@@ -17,52 +18,54 @@ module.exports = class Rubocop extends Beautifier
   }
 
   beautify: (text, language, options) ->
+    editor = atom?.workspace?.getActiveTextEditor()
+    if editor?
+      fullPath = editor.getPath()
```

Comment:
Ideally, beautifiers should not request `getActiveTextEditor` and minimize usage of Atom's APIs. The future goal is to extract these beautifiers into Node.js packages which can run without Atom.

`editor.getPath()` is unnecessary because `filePath` is passed in the `context`: https://github.com/Glavin001/atom-beautify/blob/57a64ca5093afea637974e29eee306630d8418e2/src/beautifiers/index.coffee#L346-L354

```
# Before
beautify: (text, language, options) ->

# After
beautify: (text, language, options, context) ->
   console.log("File Path", context.filePath)
```

Reply:
Note that `filePath` is optional, such as in the case where an un-saved file is being beautified. This case will need to be handled.

Reply:
Thank you for this, I copied that section from the other beautifiers. I've updated it to reflect the use of `context`.

I will have to think about how to handle a null `filePath`.

Reply:
Fixed both of these issues.

Reply:
> I copied that section from the other beautifiers

Yeah, I can imagine. Some of the other beautifiers need some love. I am trying to be more careful with PR review coming in now 😄 .